content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -74,63 +74,63 @@ For detailed PowerFlex installation procedure, see the _Dell PowerFlex Deploymen
 3. Install the SDC per the _Dell PowerFlex Deployment Guide_:
     - For environments using RPM, run `rpm -iv ./EMC-ScaleIO-sdc-*.x86_64.rpm`, where * is the SDC name corresponding to the PowerFlex installation version.
 4. To add more MDM_IP for multi-array support, run `/opt/emc/scaleio/sdc/bin/drv_cfg --add_mdm --ip 10.xx.xx.xx.xx,10.xx.xx.xx`1. Create namespace.
-   Execute `kubectl create namespace test-vxflexos` to create the `test-vxflexos` namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'test-vxflexos'.
+   Execute `kubectl create namespace vxflexos` to create the `vxflexos` namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'vxflexos'
 
 #### Create Secret
 1. Create namespace: 
-   Execute `kubectl create namespace test-vxflexos` to create the test-vxflexos namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'test-vxflexos'.
-2. Prepare the config.yaml for driver configuration.
+   Execute `kubectl create namespace vxflexos` to create the `vxflexos` namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'vxflexos'
+2. Prepare the secret.yaml for driver configuration.
 
-    Example: config.yaml
+    Example: secret.yaml
 
      ```yaml
-      # Username for accessing PowerFlex system.	
-      # Required: true
+      # Username for accessing PowerFlex system.
+      # If authorization is enabled, username will be ignored.
      - username: "admin"
-      # Password for accessing PowerFlex system.	
-      # Required: true
+      # Password for accessing PowerFlex system.
+      # If authorization is enabled, password will be ignored.
       password: "password"
       # System name/ID of PowerFlex system.	
       # Required: true
-      systemID: "ID1"
-      # REST API gateway HTTPS endpoint/PowerFlex Manager public IP for PowerFlex system.
-      # Required: true
+      systemID: "1a99aa999999aa9a"
+      # Previous names used in secret of PowerFlex system.
```

Comment:
same comment about adding clarifying comment

Reply:
Resolved in commit 7a96e18